configs/config.msi_ms7d25_ddr4

```diff
@@ -60,8 +61,4 @@ CONFIG_EDK2_DASHARO_MEMORY_CONFIG=y
 CONFIG_EDK2_BOOT_MENU_KEY=0x0015
 CONFIG_EDK2_SETUP_MENU_KEY=0x0008
 CONFIG_EDK2_RAM_DISK_ENABLE=y
-CONFIG_EDK2_CUSTOM_BUILD_PARAMS=""
 CONFIG_EDK2_CREATE_PREINSTALLED_BOOT_OPTIONS=y
-CONFIG_IPXE_ADD_SCRIPT=y
-CONFIG_IPXE_SCRIPT="3rdparty/dasharo-blobs/dasharo/dasharo.ipxe"
-CONFIG_IPXE_CUSTOM_BUILD_ID="0123456789"
```

Comment:
The IPXE Kconfig options are broken. IIRC the name changed from `IPXE_*` to `PXE_*` and olddefconfig does not recognize these. It results in PXE being included as an external binary, instead of built from iPXE source. If you regenerate the defconfigs, play pay attention to how iPXE configuration looks like in menuconfig

Reply:
I should have checked it because this did look weird.  Upstream didn't rename those options, but updated `Kconfig` in a way that `CONFIG_BUILD_IPXE` now needs to be set explicitly.  https://github.com/Dasharo/coreboot/pull/509/commits/7a9776ca0d59889e3f63d565304a7942f15985a3